oauth2client/contrib/gce.py

```diff
@@ -135,6 +136,8 @@ def _refresh(self, http_request):
                 raise HttpAccessTokenRefreshError(str(e),
                                                   status=response.status)
             self.access_token = token_content['access_token']
+            delta = datetime.timedelta(seconds=int(token_content['expires_in']))
+            self.token_expiry = delta + datetime.datetime.utcnow()
```

Comment:
That suggestion conflicts with [prior review](https://github.com/google/oauth2client/pull/473#discussion_r57394051).